batch7crud-roll4/src/main/webapp/resources/css/login.css

```diff
@@ -0,0 +1,72 @@
+body {
+    background: #d2d6de;
+}
+
+.wrapper {
+    max-width: 64em;
+    margin: 0 auto;
+    text-align: center;
+    padding: 100px 0;
+}
+
+.title {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Check this potential box model size issue [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Abox-model)